crates/utils/src/settings/structs.rs

```diff
@@ -18,6 +18,15 @@ pub struct Settings {
   pub(crate) email: Option<EmailConfig>,
   pub(crate) setup: Option<SetupConfig>,
   pub(crate) additional_slurs: Option<String>,
+  pub(crate) actor_name_max_length: Option<usize>,
+}
+
+impl Settings {
+  pub(crate) fn actor_name_max_length(&self) -> usize {
+    self
+      .actor_name_max_length
+      .unwrap_or_else(|| Settings::default().actor_name_max_length.unwrap())
+  }
 }
```

Comment:
The rest of the helper methods are not here, but in mod.rs, and they might be incorrect also.

Reply:
Fixed. I'm really surprised it let me define to impls for the same struct.